splash/har/qt.py

```diff
@@ -129,16 +133,17 @@ def reply2har(reply, include_content=False, binary_content=False):
         res["redirectURL"] = ""
 
     if include_content:
-        data = bytes(reply.readAll())
-        if binary_content:
-            res["content"]["encoding"] = "binary"
-            res["content"]["text"] = data
-            res["content"]["size"] = len(data)
-        else:
-            res["content"]["encoding"] = "base64"
-            res["content"]["text"] = base64.b64encode(data)
-            res["content"]["size"] = len(data)
-
+        content = getattr(reply, 'content', None)
+        if content is not None:
+            res["content"]["size"] = content.size()
+            bytes_array = bytes(content)
+
+            if not is_binary_string(bytes_array):
+                res["content"]["text"] = bytes_array.decode('utf8', 'replace')
```

Comment:
Giving a HAR file filled with only base64 is not the result you'd expect from a HAR file.
I think we should try to imitate chrome's behaviour with HAR files.

If you have a better way to differentiate between text and non-text, please do.
I found the above giving the most satisfying results.